src/protected_flash.rs

```diff
@@ -1135,6 +1154,27 @@ pub struct DebugSecurityPolicies {
     pub check_uuid: bool,
 }
 
+impl From<DebugSecurity> for DebugSecurityPolicies {
+    fn from(value: DebugSecurity) -> Self {
+        match value {
+            // Fixed, Disabled
+            DebugSecurity::AllDisabled =>
+                [0xffffu32, 0x0000].into(),
+            // Not fixed, Enabled
+            DebugSecurity::AllEnabled =>
+                [0x0000u32, 0xffff].into(),
```

Comment:
Please write out the settings here, one of the points of this repo is to allow users to avoid knowing what the bitfields mean unless they actually dig into the parser or to_bytes methods.

So something like
```
fn from(value: DebugSecurity) -> Self {
  use DebugSecurityPolicy::*;
  match value {
    DebugSecurity::AllEnabled => DebugSecurityPolicies {
      nonsecure_noninvasive: Enabled, // or EnabledWithDap?
      ...
    },
    ...
  }
}
```
or defining a `pub const: DebugSecurityPolicies ALL_ENABLED = ...` this way and using that.

Reply:
fixed